mix.exs

```diff
@@ -91,14 +91,13 @@ defmodule Trento.MixProject do
   defp aliases do
     [
       start: [
-        "cmd docker-compose up -d",
-        "deps.get",
-        "ecto.create",
-        "ecto.migrate",
-        "event_store.setup",
+        "install",
+        "setup",
```

Comment:
Could you restore the `cmd docker-compose up -d`?

Reply:
That's deliberate, as explained above, as it would make impossible to use `mix start` from within the container.

Reply:
some of us were relying on this, maybe we can wrap it in another task and add the docker-compose up there?

Reply:
Well, I strongly advice against developing inside a container mainly because connecting the Erlang debugger to a running containerized instance is nightmarish :/

Reply:
Oh @fabriziosestito didn't think about this, actually that would be a good "goats and cabbages" solution :sweat_smile:

Reply:
Ok, so what do we tell people? Should they just run `mix start` or should they run `docker-compose up -d` first?

Have a look at https://github.com/trento-project/web/blob/2cca604bdf9d57a3f7e43ad755987abe9aeab280/docs/development/hacking_trento.md#docker-and-docker-compose .

How do you imagine documenting the first steps? Because as it was, we gave people explicit instructions to run compose first, with an explicit command.